scripts/ingest_docs.py

```
@@ -27,6 +38,39 @@ def extract_text_from_pdf(pdf_path: Path) -> str:
+    doc = Document(docx_path)
+    text = ""
+    for paragraph in doc.paragraphs:
+        text += paragraph.text + "\n"
+    return text
+
+def extract_text_from_html(html_path: Path) -> str:
+    if not BeautifulSoup:
+        logger.error("beautifulsoup4 not installed. Cannot read HTML files.")
+        sys.exit(1)
+
+    with open(html_path, 'r', encoding='utf-8') as f:
```

Ruff (0.14.10)

Unnecessary mode argument

Remove mode argument

(UP015)